tests/python/pytest_opinfos.py

```diff
@@ -1103,6 +1105,33 @@ def torch_reshape_sym_fn(input_tensor, output_shaped_tensor):
 )
 tensor_creation_ops.append(uniform_opinfo)
 
+matmul_ops = []
+
+
+def matmul_wrapper(fd, data_type, a, b):
+    t1 = fd.ops.matmul(a, b)
+    t2 = fd.ops.cast(t1, dtype=DataType.Half)
+    return t2
+
+
+matmulfp16_opinfo = OpInfo(
+    lambda fd: partial(matmul_wrapper, fd, DataType.Half),
+    "matmul_float16",
+    dtypes=(torch.float16,),
```

Comment:
Are we able to access an `nvfuser.Tensor`'s dtype from python? If so then you could just add the dtypes in this list and then use `a.dtype()` instead of an explicit parameter right? This is a minor nitpick of course.

Reply:
Unfortunately, we can't access the dtype. I'd like to expose that, but I think that should be a separate PR.